taichi/python/export_lang.cpp

```diff
@@ -896,7 +999,7 @@ void export_lang(py::module &m) {
 
   m.def(
       "create_kernel",
-      [&](const std::function<void()> &body, const std::string &name,
+      [&](const std::function<void(Kernel *)> &body, const std::string &name,
```

Comment:
Off-topic comment: is this function overriding the other `create_kernel` above?

Reply:
Yes, I think so. I don't know why it is exported twice. In the future, maybe we can remove one in which the `py::gil_scoped_release` isn't used.

Reply:
> Off-topic comment: is this function overriding the other `create_kernel` above?

Yes, I think so. I don't know why it is exported twice. In the future, maybe we can remove one in which the `py::gil_scoped_release` isn't used.